tests/integration/integration_test.go

```diff
@@ -986,7 +986,6 @@ func TestDestroyStackRef(t *testing.T) {
 
 	e.RunCommand("pulumi", "up", "--skip-preview", "--yes")
 
-	e.CWD = os.TempDir()
```

Comment:
Oh this was by design so that destroy didn't need a Pulumi.yaml file. We should see if we can maintain that support for now.